frontend/app/actions/TouchpointActionRefiners.scala

```diff
@@ -26,18 +26,6 @@ class TouchpointActionRefiners(authenticationService: AuthenticationService, tou
   implicit private val ec = executionContext
   implicit private val tpbs = touchpointBackends
 
-  private def getContributorRequest[A](request: AuthRequest[A]): Future[String \/ Option[SubReqWithContributor[A]]] = {
```

Comment:
I removed a fair bit of code related to checking if a partner has used their free ticket allocation.  This is because it has been broken for a long time and is a known issue.